crates/sui-sdk/src/apis.rs

```diff
@@ -120,10 +388,28 @@ impl ReadApi {
             .await?)
     }
 
+    /// Return the total number of transaction blocks, or an error upon failure.
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use sui_sdk::SuiClientBuilder;
+    ///
+    /// #[tokio::main]
+    /// async fn main() -> Result<(), anyhow::Error> {
+    ///     let sui = SuiClientBuilder::default().build_localnet().await?;
+    ///     let total_transaction_blocks = sui
+    ///         .read_api()
+    ///         .get_total_transaction_blocks()
+    ///         .await?;
+    ///     Ok(())
+    /// }
+    /// ```
     pub async fn get_total_transaction_blocks(&self) -> SuiRpcResult<u64> {
         Ok(*self.api.http.get_total_transaction_blocks().await?)
     }
 
+    /// Return a [SuiTransactionBlockResponse] based on the given [TransactionDigest], or an error upon failure.
```

Comment:
A brief description of what a `SuiTransactionBlockResponse` contains wouldn't hurt here, e.g.:

```rust
/// Return a transaction and its effects in a [SuiTransactionBlockResponse] based on its 
/// [TransactionDigest], or an error upon failure.
```